docs/docs/02-guide/06-tokenfactory/01-tokenfactory.md

```diff
@@ -0,0 +1,622 @@
+
+**Proto Adjustments**
+
+In `proto/tokenfactory/tokenfactory/tx.proto`, remove the `DeleteDenom` RPC method and the associated message types.
+
+**Client Updates**
+
+Navigate to the client in `x/tokenfactory/client` and make these changes:
+
+- Remove `TestDeleteDenom()` from `tx_denom_test.go`.
+- Eliminate `CmdDeleteDenom()` from `tx_denom.go`.
+- In `tx.go`, delete the line referencing the delete command.
+
+**Keeper Modifications**
+
+In `denom_test.go`, remove `TestDenomRemove()`.
+Delete `RemoveDenom()` from `denom.go`.
+Exclude `TestDenomMsgServerDelete()` and `DeleteDenom()` functions from `msg_server_denom_test.go` and `msg_server_denom.go`, respectively.
+
+**Types Directory Changes**
+
+- Update `codec.go` to remove references to `MsgDeleteDenom`.
+- Remove `TestMsgDeleteDenom_ValidateBasic()` from `messages_denom_test.go`.
+- Eliminate all references to `MsgDeleteDenom()` in `messages_denom.go`.
+
+After making these changes, commit your updates:
+
+```bash
+git add .
+git commit -m "Remove the delete denom functionality"
+```
+
+This concludes the second chapter, setting a solid foundation for your token factory module. In the next chapter, we'll delve into implementing the application logic that will bring your token factory to life.
+
+## Chapter 3: Implementing Core Functionality in Your Token Factory
+
+Having disabled the deletion of denoms, we now turn our attention to the heart of the token factory module: defining the structure of new denoms and implementing their creation and update logic.
+
+**Proto Definition Updates**
+
+Start by defining the structure of a new token denom in `proto/tokenfactory/tokenfactory/tx.proto`.
+
+For `MsgCreateDenom`:
+
+- Remove `int32 supply = 8;` and adjust the field order so `canChangeMaxSupply` becomes the 8th field.
+
+Resulting `MsgCreateDenom` message:
+
+```proto
+message MsgCreateDenom {
+  string owner = 1;
+  string denom = 2;
+  string description = 3;
+  string ticker = 4;
+  int32 precision = 5;
+  string url = 6;
+  int32 maxSupply = 7;
+  bool canChangeMaxSupply = 8;
+}
+```
+
+For `MsgUpdateDenom`:
+
+- Omit `string ticker = 4;`, `int32 precision = 5;`, and `int32 supply = 8;`, and reorder the remaining fields.
+
+Resulting `MsgUpdateDenom` message:
+
+```proto
+message MsgUpdateDenom {
+  string owner = 1;
+  string denom = 2;
+  string description = 3;
+  string url = 4;
+  int32 maxSupply = 5;
+  bool canChangeMaxSupply = 6;
+}
+```
+
+### Client Logic
+
+In the `x/tokenfactory/client/cli/tx_denom.go` file, update the client application logic.
+
+**For `CmdCreateDenom`:**
+
+- Adjust the number of arguments from 8 to 7, removing references to the supply argument, and update the usage descriptions.
```

Comment:
maybe it is better to add an example below the `Client Logic` explanation

Reply:
This is removing one line and editing another, the `supply` - does this really need the code?